docs/develop/create-sso-office-add-ins-aspnet.md

```diff
@@ -16,9 +16,9 @@ This article walks you through the process of enabling single sign-on (SSO) in a
 
 ## Prerequisites
 
-* The latest available version of Visual Studio 2017 Preview.
+* The latest available version of Visual Studio 2017.
 
-* Office 2016, Version 1708, build 8424.nnnn or later (the Office 365 subscription version, sometimes called “Click to Run”). You might need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1).
+* Office 365 (subscription version, also called “Click to Run”). Latest monthly version and build from the Insiders channel. You need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1). Note: when a version/build graduates to the production semi-annual channel, support for preview features, including SSO, is turned off for that version/build.
```

Comment:
Instead of "version/build", would it still be accurate to say "version or build"? Relatedly, does a build ever get pushed to production without a version update? I'm wondering if we can remove "build" from this section to make it less jargon-y.

Reply:
We have to use "build" because flights are turned on an off on builds. Also, the yymm (like 1902) version is not specific enough. However, in the note part, I can replace "version/build" with just "build". And, yes, a particular version-build combination does get pushed to the semi-annual (prod) channel without a change in version. I'll make your other suggestions.